Rethinking_2/Chp_02.ipynb

```diff
@@ -61,7 +61,7 @@
     "$$Pr(w \\mid n, p) =  \\frac{n!}{w!(n − w)!} p^w (1 − p)^{n−w}$$\n",
```

Comment:
<p>I suppose this cell should output a graph?</p><p>Also, can you split up this cell between the model part and the graph part please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Hi, I hope I will get all the parts since all of your comments point to the same place. Which plot is missing? Regarding the split of the model and plot -- I was thinking about that but at the end preferred one for loop with this over two blocks of code, two for loops. I can redo it if you insist.

Reply:
<p>Ooh no, you're right, that's better like that, with just one for loop.</p><p>Regarding my other comments, you should check them out with the ReviewNB link that's automatically generated above -- GitHub is really for reviewing NBs</p>

Reply:
<p>Oh, sorry, I see now :), I'm still pretty new to github (I mean, I used it for myself but never really with anyone else, so I'd be happy for any tips)</p><p> </p><p>I believed I fixed all of the issues now.</p>

Reply:
No problem, we've all been beginners once 😉
As a general advice, I'd suggest using more explicit PR titles, branch names and commit messages -- it will make life easier for reviewers and future you :)